README.adoc

```diff
@@ -405,9 +405,9 @@ all of its lines.
 Uncomment all the lines in the [hotspot file=0]`openapi.yaml` file.
 
 openapi.yaml
-[source, text, linenums, role="code_column"]
+[source, text, linenums, role='code_column']
```

Comment:
Instead of `text`, the file needs to be formatted as a `yaml` file

Reply:
Line 393, hotspot the file reference

Reply:
Based on the guide's original format, creating a hotspot for the file seems excessive since the arrowhead below references the same file.

Reply:
> Instead of text, the file needs to be formatted as a yaml file

Unresolved

Reply:
Not sure what the guide developer's intention was to format as text, instead of a `yaml` file. Leaving it for now.